iron-selection.html

```diff
@@ -32,7 +32,7 @@
      * the selected item or undefined if there is no selection.
      */
     get: function() {
-      return this.multi ? this.selection : this.selection[0];
+      return this.multi ? this.selection.slice() : this.selection[0];
```

Comment:
Yes. I believe returning a copy will guarantee data integrity. If the reference is returned, data changed on that object, then someone uses the element accessors to change data in the component, then unexpected behavior results. This way, the element's selection integrity is maintained.